weaviate_cli/managers/data_manager.py

```diff
@@ -48,7 +53,7 @@ def __import_json(
                         for prop in properties:
                             prop_name = PROPERTY_NAME_MAPPING.get(prop.name, prop.name)
                             if prop_name in obj:
-                                added_obj[prop_name] = self.__convert_property_value(
+                                added_obj[prop.name] = self.__convert_property_value(
```

Comment:
typo here? Shouldn't it be prop_name?

Reply:
No, it's on purpose. Here prop is the property obtained from the collection [properties configuration](https://github.com/weaviate/weaviate-cli/blob/main/weaviate_cli/managers/collection_manager.py#L120-L134). Then, to be able to use the contextionary vectorizer (it requires the properties names in camelCase format, it won't accept a string separated by underscore) we have the PROPERTY_NAME_MAPPING that maps the property from camelCase to the underscore one.
Therefore, when we add the object we need to use the prop.name (which is the same name used when defining the properties in class creation). As it was before, it was skipping all properties which had camelCase and it wouldn't add any object for those, that's why I swapped also the content in PROPERTY_NAME_MAPPING.